projects/demo/src/modules/experimental/chip/examples/2/index.less

```diff
@@ -0,0 +1,11 @@
+:host {
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+}
+
+section {
```

Comment:
Let's make this [example](https://taiga-ui--pr6076-chip-fmxei108.web.app/experimental/chip#sizes) more mobile-friendly:

<img width="691" alt="iphone12" src="https://github.com/taiga-family/taiga-ui/assets/35179038/dfca922d-111e-478d-9ce2-6a7e607b4ba9">

Reply:
`flex-wrap: wrap` ?